feat(docs): add feedback searchable properties (#8599)
getsentry/sentry-docs: src/docs/product/reference/search/searchable-properties/user-feedback.mdx

---
title: User Feedback Properties
sidebar_order: 40
description: "Learn more about searchable user feedback properties."
---

<Include name="searchable-properties/intro-for-user-feedback.mdx" />

## Searchable Properties

Below is a list of keys and tokens that can be used in the user feedback search.

<DefinitionList>

### `assigned`

> Returns issues assigned to the defined user(s) or team(s). Values can be a user ID (your email address), `me` for yourself, `none` for no assignee, `my_teams` or `#team-name` for teams you belong to.

Reviewer: ```suggestion
Returns user feedback submissions assigned to the defined user(s) or team(s). Values can be a user ID (your email address), `me` for yourself, `none` for no assignee, `my_teams` or `#team-name` for teams you belong to.
```

- **Type:** team or org user

### `browser.name`

Name of the browser.

- **Type:** string

### `device.brand`

Brand of the device.

- **Type:** string

### `device.family`

Family of the device. Typically, the common part of a model name across generations. For example, iPhone, Samsung Galaxy.

- **Type:** string

### `device.model_id`

Internal hardware revision to identify the device exactly.

- **Type:** n/a

### `device.name`

Name of the device.

- **Type:** string

### `dist`

Distinguishes build or deployment variants of the same release of an application. For example, the dist can be the build number of an XCode build or the version code of an Android build.

- **Type:** string

### `environment`

The environment that the event was first seen in.

- **Type:** string

### `id`

The feedback ID.

- **Type:** UUID

### `is`

> The properties of an issue. Values can be: `unresolved`, `resolved`, `assigned`, `unassigned`, `linked`, or `unlinked`. The `linked` and `unlinked` values return issues based on whether they are linked to an external issue tracker or not.

Reviewer: ```suggestion
The properties of a user feedback submission. Values can be: `unresolved`, `resolved`, `assigned`, `unassigned`, `linked`, or `unlinked`. The `linked` and `unlinked` values return user feedback submissions based on whether they are linked to an external issue tracker or not.
```

Reviewer: To be consistent w/ product docs

- **Type:** status

### `level`

> The severity of the user feedback. If the feedback came in through the crash-report modal, then the level would be `error`; otherwise, the level is `info`.

Reviewer: ```suggestion
The severity of the user feedback for submissions that came in through the crash-report modal and have an error attached.
```

Reviewer: > otherwise, the level is `info`.
-
Do we reflect anywhere that the feedback submission from our user feedback widget is "info" level? I don't think that's part of the UI anymore :/ Perhaps we can still leave the note that submissions from the user feedback widget should have an info level in order for people to filter their results for just that.

Author: nope, we don't indicate the "info" level anywhere

Reviewer: Ok I suggested something else!

- **Type:** string

### `os.name`

The name of the operating system.

- **Type:** string

### `sdk.name`

Name of the Sentry SDK that sent the event.

- **Type:** string

### `sdk.version`

Version of the Sentry SDK that sent the event.

- **Type:** string

### `timestamp`

The finish timestamp of the transaction. Returns events with matching datetime.

- **Type:** datetime

### `transaction`

The error or transaction name identifier.

- **Type:** string

> ### `user.email`

Reviewer: We can't search by user email if the email was entered by the user inside the feedback widget. I think it's important to note that this email search is specifically if we get the user email from the `Sentry.setUser` SDK option as a tag. cc/ @bruno-garcia

Author: adjusted wording but not sure the best way to phrase this

Reviewer: me too, that's why i tagged bruno in hopes he would have a good way to convey this 😭

---
title: User Feedback Properties
sidebar_order: 40
description: "Learn more about searchable user feedback properties."
redirect_from:
  - /product/sentry-basics/search/searchable-properties/user-feedback/
---

<Include name="searchable-properties/intro-for-user-feedback.mdx" />

## Searchable Properties

Below is a list of keys and tokens that can be used in the user feedback search.

<DefinitionList>

### `assigned`

Returns user feedback submissions assigned to the defined user(s) or team(s). Values can be a user ID (your email address), `me` for yourself, `none` for no assignee, `my_teams` or `#team-name` for teams you belong to.

- **Type:** team or org user

### `browser.name`

Name of the browser.

- **Type:** string

### `device.brand`

Brand of the device.

- **Type:** string

### `device.family`

Family of the device. Typically, the common part of a model name across generations. For example, iPhone, Samsung Galaxy.

- **Type:** string

### `device.model_id`

Internal hardware revision to identify the device exactly.

- **Type:** n/a

### `device.name`

Name of the device.

- **Type:** string

### `dist`

> Distinguishes build or deployment variants of the same release of an application. For example, the dist can be the build number of an XCode build or the version code of an Android build.

Reviewer 2: tiny nit: `Xcode` instead of `XCode` :)

Author: ahh, this is in a couple places in the docs too. i'll fix those other places as well

- **Type:** string

### `id`

The feedback ID.

- **Type:** UUID

### `is`

The properties of a user feedback submission. Values can be: `unresolved`, `resolved`, `assigned`, `unassigned`, `linked`, or `unlinked`. The `linked` and `unlinked` values return user feedback submissions based on whether they are linked to an external issue tracker or not.

- **Type:** status

### `level`

The severity of the user feedback submissions. If the feedback came in through the crash-report modal then the level is tied to the error experienced by the end-user. For feedback from the user feedback widget, the level is `info`.

- **Type:** string

### `os.name`

The name of the operating system.

- **Type:** string

### `sdk.name`

Name of the Sentry SDK that sent the event.

- **Type:** string

### `sdk.version`

Version of the Sentry SDK that sent the event.

- **Type:** string

### `timestamp`

The finish timestamp of the transaction. Returns events with matching datetime.

- **Type:** datetime

### `transaction`

The error or transaction name identifier.

- **Type:** string

### `url`

The URL of the page that the feedback is triggered on.

- **Type:** string

### `user.email`

> User's email addrss. This email search applies only if Sentry receives the user email from the `Sentry.setUser` SDK option as a tag.

Reviewer 3: ```suggestion
User's email address. This email search applies only if Sentry receives the user email from the `Sentry.setUser` SDK option as a tag.
```

> User's email address. This email search applies only if Sentry receives the user email from the `Sentry.setUser` SDK option as a tag.

Reviewer 4: ```suggestion
The user's email address only if Sentry received it from the `Sentry.setUser` SDK option as a tag. This property does not apply to emails entered inside the user feedback widget, which are not searchable.
```

I think we should call out that it doesn't apply to emails from the widget. Maybe Bruno has a better way to word this

Author: updated, cc @bruno-garcia if he has any suggestions

Reviewer 5: @cmanallen is working on this right now through https://github.com/getsentry/sentry/issues/59928 so we might have the right wording based on his implementation

Reviewer 5: We can also unblock this and make another PR with the updated version
